src/main/webapp/dev/js/main/instructorFeedbackEdit.js

```diff
@@ -1217,6 +1223,7 @@ $(document).ready(() => {
 window.updateConstSumPointsValue = updateConstSumPointsValue;
 window.addConstSumOption = addConstSumOption;
 window.removeConstSumOption = removeConstSumOption;
+window.toggleConstSumOptionsRadioButton = toggleConstSumOptionsRadioButton;
```

Comment:
Is this used?

Reply:
Oops. Sorry, it is not used. Removing it.